### DISCUSSION TITLE
Has NVIDIA's decision to start open sourcing their graphics drivers/modules changed your mind on future GPU purchases?

### DISCUSSION THREAD

**Discussion by ghtesting2020** [NVIDIA/open-gpu-kernel-modules#200] [Polls]
I had a terrible time with my Optimus laptop and my friend with their RTX 2080 Super and we both said we will go AMD next time due to their friendliness with Linux. However, now that NVIDIA has _started_ the process to open source their graphics drivers/modules we will probably stick with NVIDIA depending on how much further they go with Linux support. Currently this module is a great start but it is not enough. Intel Arc is also supposed to be super friendly with Linux.

**Comment by sTiKyt** [comment]
Still worried about all of this "there is no git history" nonsense but we'll see what happens

**Reply by SirObby** [reply]
keyword, 'may', they are merging PRs directly from GitHub.

**Comment by yoshimo** [comment]
It is too early to tell.
Firmware for maxwell that allows reclocking is missing and we don't have a proper user space yet.
This is just the beginning and I also don't need a new card anytime soon.

**Reply by Evernow** [reply]
> maxwell 

This post is in regards to future GPU purchases, very few people are buying these old GPUs nowadays.

**Reply by sTiKyt** [reply]
Not sure about official stats but around where I live there are still tons of people who can't afford to buy new gpu, prices are crazy and war only made it worse

**Comment by bgkillas** [comment]
no good mesa support makes this meaningless to me

**Reply by YusufKhan-gamedev** [reply]
Working on it!!! Wait pls, I sent a MR a while ago to libdrm that adds support there though...

**Reply by mobin-2008** [reply]
https://gitlab.freedesktop.org/mesa/drm/-/merge_requests/238

**Comment by potatogod3000** [comment]
Need mesa support to mature in future or atleast open-sourcing the NVIDIA Userspace driver for me to buy NVIDIA GPU again.

**Comment by Danny3** [comment]
One small good deed is not enough to repay lots of years of bad deeds!

If Nvidia really goes the open source way, without tricks like moving pretty much everything in the closed source firmware, starts using Mesa and brings something new to the table like, HDR support, I might reconsider continuing to buy AMD GPUs, but until then, I'm not interested.

**Comment by es20490446e** [comment]
Only if it runs flawlessly on open source drivers... something that has happened for the other two for a decade...

**Comment by HPlinko** [comment]
Sorry, but until the open-source drivers are as good or even better than the proprietary ones nvidia's still not getting my money.

**Comment by Madouura** [comment]
I put other, since it's just too early to tell, and depends on other factors.
That said, if by say a generation or two ahead in NVIDIA gpus they have superior hardware, and the software is up-to-par with current AMD software in Linux, I may indeed buy NVIDIA instead.

**Comment by knoopx** [comment]
So far, the released code makes no difference for the end user/consumer market, so I can't see how this might influence anyone to make a choice.

**Reply by sTiKyt** [reply]
> So far, the released code makes no difference for the end user/consumer market, so I can't see how this might influence anyone to make a choice.

To some it's about ideology, principles

**Comment by mobin-2008** [comment]
Hello.
I've been trying to sell my GTX 1650 for a while and buy an RX 6600 graphics card, but Nvidia's decision changed my vision!
Nouveau developers and I have been waiting for about a decade (since the release of Maxwell's second generation) for NVIDIA to move in the direction of the open source community.
Of course, over the years, NVIDIA has made sporadic efforts to improve the situation. NVIDIA, for example, released a number of firmwares for newer graphics cards from Maxwell's second generation, all of which, unfortunately, did not have a PMU. (PMU is a firmware that allows you to control the temperature, fan speed and clock of the graphics chip and graphics memory)
This made me totally disappointed that NVIDIA would never be right. But this recent decision has given me more hope for the future.
I also recently learned that someone is building a program that can use nvidia-open-gpu via libdrm:
https://gitlab.freedesktop.org/mesa/drm/-/merge_requests/238

This promises a good future for those who like to use their NVIDIA graphics card well and not get involved with proprietary drivers.

I look forward to seeing what happens in the future ...

**Comment by YusufKhan-gamedev** [comment]
My choice of " No, this is not enough and I will go AMD Radeon GPU next time " was not because of any open sourcing effort, imho this ogk driver is enough as someone sooner or later will write open source userland drivers meaning that nvidia doesnt need to put any work into open sourcing the userland. The reason I dont buy from NVIDIA is because NVIDIA GPUs though they perform well usually are a worse quality failing more than the Radeon ones in my experience.

**Comment by edisionnano** [comment]
The lack of Pascal and Maxwell support is rather concerning for me personally and so is the radio silence regarding PMU firmware for these generations of graphics cards. My next purchase wil, hopefully, be a Radeon 6600xt; probably the best Linux gpu. It will take a while for these drivers to mature and nouveau devs to make use of them and make a working upstream kernel driver and even if that happens we don't know if nvidia will contribute to an upstream driver; if not support will be slow and lucklaster, AMD does and hardware usually works day 1, plug and play. Then we have the userspace, RadeonSI is an excelent opengl driver that AMD, Valve and others contribute to, miles better than the prop gl driver of AMD, also CPU overhead has been much lower than nvidia in my experience. Then you have RADV, the Vulkan driver used in the steam deck that valve carries which is just so much better for D3D12 games. I'm sorry but I have had a bad experience with my 1050ti so my next card is definately going to be AMD. If in 5-7 NVIDIA open source support is in a good place, officially supported by nvidia, aided by Valve devs etc then the card after the next one may as well be from NVIDIA.
